app/services/api/v2/suggestions_service.rb

```diff
@@ -1,22 +1,120 @@
 module Api
   module V2
```

Comment:
Referencing the api version for this global service makes 0 sense since no V1 api exists and this is global